jobs/s-bandwidth.yaml

```diff
@@ -0,0 +1,21 @@
+suite: s-bandwidth
+testcase: s-bandwidth
+category: benchmark
+
```

Comment:
is there any disc needed for this test, if yes, can add extra info like disk: 1HDD? and which type of fs is supposed to be tested?

Reply:
added the requested info